### PR TITLE
Allow installing using a registry stackname

### PR DESCRIPTION
**Pull request by kke**

### CODE REVIEW

**Comment by SpComb** [comment]
WFM

```
$ curl -s http://192.168.66.102:80/ | grep '<meta name="generator"'
$ bundle exec bin/kontena stack install --name wordpress-green --deploy terom/wordpress:4.6.1+mariadb5.5
 [done] Creating stack wordpress-green      
 [done] Deploying stack wordpress-green     
$ curl -s http://192.168.66.102:80/ | grep '<meta name="generator"'
<meta name="generator" content="WordPress 4.6.1" />
```

**Comment by jnummelin** [comment]
DNWFM, or something:
```
$ bundle exec bin/kontena stack install --name redis --deploy jakolehm/redis-sentinel:0.1.0
 [error] File jakolehm/redis-sentinel:0.1.0 does not exist
```

File: cli/lib/kontena/cli/stacks/install_command.rb [hunk 1]
```diff
@@ -17,8 +17,15 @@ class InstallCommand < Kontena::Command
     requires_current_master_token
 
     def execute
-      require_config_file(filename)
-      stack = stack_from_yaml(filename)
+
+      if !File.exist?(filename) && filename =~ /\A\w+\/\w+(?::.*)?\z/
```
Comment (SpComb) [inline cli/lib/kontena/cli/stacks/install_command.rb]: This regexp is too strict, the `\w` does not match e.g. `redis-sentinel`

Comment (kke) [inline cli/lib/kontena/cli/stacks/install_command.rb]: regex eased

**Pull request merged**